src/engraving/layout/layoutsystem.cpp

```diff
@@ -168,14 +168,14 @@ System* LayoutSystem::collectSystem(const LayoutOptions& options, LayoutContext&
             system->layout2(ctx);         // compute staff distances
             return system;
         }
+
         // check if lc.curMeasure fits, remove if not
         // collect at least one measure and the break
-
-        // acceptanceRange slightly larger than 1 allows systems to be initially slightly larger than the target width
-        // and be justified by squeezing rather than stretching. However, we must first make sure that the system *can*
-        // be squeezed. I'm temporarily rolling back the idea (still a bit too risky in some edge cases) [M.S.]
-        double acceptanceRange = 1;
-        bool doBreak = (system->measures().size() > 1) && ((curSysWidth + ww) > systemWidth * acceptanceRange);
+        double acceptanceRange = 0.3 * system->squeezableSpace();
```

Comment:
Where does the 0.3 come from?

Reply:
Nothing, it just looks good. May be worth a comment though, thanks.